gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/CseValidHandler.java

```diff
@@ -321,6 +321,32 @@ private double computeShiftValue(TTimestampWrapper timestampWrapper) {
 
     private void runDichotomyForExportCorner(TTimestampWrapper timestampWrapper, CseValidRequest cseValidRequest, TcDocumentTypeWriter tcDocumentTypeWriter, String jsonCracUrl, String raoParametersURL, Network network, CseCracCreationContext cracCreationContext) {
         DichotomyResult<RaoResponse> dichotomyResult = dichotomyRunner.runExportCornerDichotomy(timestampWrapper, cseValidRequest, jsonCracUrl, raoParametersURL, network);
-        // TODO
+        if (dichotomyResult != null && dichotomyResult.hasValidStep()) {
+            String raoResultFileUrl = dichotomyResult.getHighestValidStep().getValidationData().getRaoResultFileUrl();
+            RaoResult raoResult = fileImporter.importRaoResult(raoResultFileUrl, cracCreationContext.getCrac());
+            TLimitingElement tLimitingElement = LimitingElementHelper.getLimitingElement(raoResult, cracCreationContext, network);
+            BigDecimal value = computeValueForExportCorner(timestampWrapper, dichotomyResult)
+                    .map(Math::round)
+                    .map(BigDecimal::valueOf)
+                    .orElse(BigDecimal.ZERO);
```

Comment:
Use a double and manage the BigDecimal conversion directly in the tcDocumentTypeWriter. That is unnecessary conversion in a functional point of view